doc/manual/workflow-tips.rst

```diff
@@ -75,7 +75,7 @@ issuing the command::
 
 If you further add the following to your ``.juliarc.jl`` file ::
 
-    isinteractive() && isfile("_init.jl") && require("_init.jl")
+    isfile("_init.jl") && include(joinpath(pwd(), "_init.jl"))
```

Comment:
include is relative to current file. so at least this makes it consistent with the `isfile` check. I'm not sure which one (relative to current file or PWD) is the goal here.